Vagrantfile

```diff
@@ -0,0 +1,32 @@
+Vagrant.configure("2") do |config|
+config.vm.box = "hashicorp/bionic64"
```

Comment:
Your indentation in this file is slightly off, though it doesn't affect the functionality

Lines 2 and 4 should be indented another level.

Lines 13,14 should be indented another 2 spaces.

Line 22 should be indented less and the following lines should be indented more